.github/workflows/cypress.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Install dependencies
-        run: npm i && npm i -g cypress@9.0.0
+        run: npm i && npm i -g cypress@11.2.0
```

Comment:
Is there a specific reason to install Cypress globally instead of using the local version?

Reply:
We set env vars from repo secrets for a twitter API key and secret. So, directly calling the cypress command in workflow instead of using npm scripts. see [here](https://github.com/10up/autoshare-for-twitter/blob/develop/.github/workflows/cypress.yml#L43)

Reply:
@Sidsector9 I tried a different approach to setup cypress env variables and removed global cypress installation.